stock_indicators/indicators/common/helpers.py

```
@@ -9,30 +9,55 @@
+                raise TypeError("remove_periods must be an integer")
+            if remove_periods < 0:
+                raise ValueError("remove_periods must be non-negative")
             return super().remove_warmup_periods(remove_periods)
 
-        removed_results = CsIndicator.RemoveWarmupPeriods(CsList(self._get_csdata_type(), self._csdata))
-        return self.__class__(removed_results, self._wrapper_class)
+        try:
+            removed_results = CsIndicator.RemoveWarmupPeriods(CsList(self._get_csdata_type(), self._csdata))
+            return self.__class__(removed_results, self._wrapper_class)
+        except Exception as e:
+            raise RuntimeError(f"Failed to remove warmup periods: {e}") from e
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)